test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/task/PostgreSQLIncrementTask.java

```diff
@@ -73,8 +73,9 @@ public void run() {
     private Object insertOrder() {
         ThreadLocalRandom random = ThreadLocalRandom.current();
         String status = 0 == random.nextInt() % 2 ? null : "中文测试";
-        Object[] orderInsertDate = new Object[]{KEY_GENERATE_ALGORITHM.generateKey(), random.nextInt(0, 6), status, PipelineCaseHelper.generateJsonString(2)};
-        String insertSQL = String.format("INSERT INTO %s (order_id,user_id,status,t_json) VALUES (?, ?, ?, ?)", getTableNameWithSchema(orderTableName));
+        String jsonString = PipelineCaseHelper.generateJsonString(2);
+        Object[] orderInsertDate = new Object[]{KEY_GENERATE_ALGORITHM.generateKey(), random.nextInt(0, 6), status, jsonString, jsonString.getBytes()};
```

Comment:
1, `jsonString.getBytes()`, could we support Unicode character?

2, It's better to make `PipelineCaseHelper.generateJsonString(2)` could generate Unicode characters.

Reply:
I find data consistency check failed, i need takes a little time to check